### PR TITLE
SlurmGCP. Rework logging

### PR DESCRIPTION
**Pull request by mr0re1**
* Don't initialize file-specific logger, instead use "root logger" configured with proper destination file;
* Don't pass `log` as an argument to the `setup_network_storage.py` functions, use "root logger" instead;
* Stop using excessive `argparse.ArgumentParser` initialization since none of scripts define `__doc__`.

### CODE REVIEW

File: community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/util.py [hunk 1]
```diff
@@ -580,9 +571,8 @@ def log_api_request(request):
     log.debug(f"{inspect.stack()[1].function}:\n{pretty_req}")
 
 
-def handle_exception(exc_type, exc_value, exc_trace):
+def _handle_exception(exc_type, exc_value, exc_trace):
     """log exceptions other than KeyboardInterrupt"""
-    # TODO does this work?
```
Comment (harshthakkar01) [inline community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/util.py:585]: Was this here to verify something ?

File: community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup_network_storage.py [hunk 2]
```diff
@@ -30,6 +31,8 @@
 from more_executors import Executors, ExceptionRetryPolicy
 
 
+log = logging.getLogger()
```
Comment (harshthakkar01) [inline community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup_network_storage.py:34]: for this root logger, what would be destination file ?

File: community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/util.py (hunk 1 above)
Comment (mr0re1) [reply community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/util.py:585]: Yep, verified.

File: community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup_network_storage.py (hunk 2 above)
Comment (mr0re1) [reply community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup_network_storage.py:34]: It depends on what "main" uses it.

**Review by harshthakkar01 — APPROVED** [review]

**Pull request merged**